### PR TITLE
Update styles on responsive tables

### PR DESCRIPTION
**Pull request by micahchiang**
## Description
Per design feedback, several styles have been updated on the view-payments responsive tables. Based on [these mockups](https://xd.adobe.com/view/5f609803-b045-428a-510e-1754e404af3e-2951/screen/6a2de0f2-3a3b-4da0-b252-471e5cdcb7e9/specs/)

## Testing done
- local

## Screenshots
- vertical aligned elements:
Screen Shot 2020-07-23 at 3 19 37 PM

- bottom border on display heading:
Screen Shot 2020-07-23 at 3 19 06 PM

- added utility to class to force font-family `Bitter`:
Screen Shot 2020-07-23 at 3 20 06 PM

## Acceptance criteria
- [x] styles have been updated and accurately reflect the mockups. 

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/disability-benefits/view-payments/components/responsive-table/ResponsiveTable.jsx [hunk 1]
```diff
@@ -1,6 +1,5 @@
 import React, { Component } from 'react';
 import classNames from 'classnames';
-import './ResponsiveTable.scss';
```
Comment (micahchiang) [inline src/applications/disability-benefits/view-payments/components/responsive-table/ResponsiveTable.jsx:3]: imported this file at the app level so it's available everywhere

File: src/applications/disability-benefits/view-payments/components/view-payments-lists/payments/Payments.jsx [hunk 2]
```diff
@@ -70,7 +70,7 @@ class PaymentsReceived extends Component {
       tableContent = (
         <>
           {this.props.textContent}
-          <p className="vads-u-font-size--lg">
```
Comment (micahchiang) [inline src/applications/disability-benefits/view-payments/components/view-payments-lists/payments/Payments.jsx:73]: force bitters

File: src/applications/disability-benefits/view-payments/sass/ResponsiveTable.scss [hunk 3]
```diff
@@ -1,20 +1,27 @@
+$responsive-table-width-medium: 600px;
```
Comment (micahchiang) [inline src/applications/disability-benefits/view-payments/sass/ResponsiveTable.scss:1]: Just put our custom breakpoint in a sass variable.

File: src/applications/disability-benefits/view-payments/sass/ResponsiveTable.scss [hunk 4]
```diff
@@ -33,6 +40,10 @@ table.responsive td {
   table.responsive tr {
     display: block;
     border-bottom: 1px solid #757575;
+
+    &:first-child {
+      border-top: 1px solid #757575 !important;
```
Comment (micahchiang) [inline src/applications/disability-benefits/view-payments/sass/ResponsiveTable.scss:45]: This is the border between `Displaying 1-6 of 16`. Mockups indicate equal spacing between that heading and the first element in the table. This seems to make the most sense.

**Review by Mottie — APPROVED** [review]

**Pull request merged**